src/work-allocation-2/containers/work-case-list-wrapper/work-case-list-wrapper.component.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { CdkTableModule } from '@angular/cdk/table';
+import { ChangeDetectorRef} from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AlertService, LoadingService, PaginationModule } from '@hmcts/ccd-case-ui-toolkit';
+import { ExuiCommonLibModule, FeatureToggleService} from '@hmcts/rpx-xui-common-lib';
+import { of } from 'rxjs';
+import { SessionStorageService } from '../../../app/services';
+import { WorkAllocationComponentsModule } from '../../components/work-allocation.components.module';
+import { Case } from '../../models/cases';
+import { InfoMessageCommService, WorkAllocationCaseService, WorkAllocationFeatureService } from '../../services';
+import { getMockCases, MockRouter } from '../../tests/utils.spec';
+import { MyCasesComponent } from '../my-cases/my-cases.component';
+import { WorkCaseListComponent } from '../work-case-list/work-case-list.component';
+import { WorkCaseListWrapperComponent } from './work-case-list-wrapper.component';
```

Comment:
_Nitpick_:
I think these imports might need organising.

Reply:
Reorganised

Reply:
Not quite the order I had in mind but, as discussed on Slack, you said the imports were being weird and you were getting compilation errors. This was only a _nitpick_ anyway.